lib/prisma.ts

```
@@ -0,0 +1,18 @@
+import { PrismaClient } from '@prisma/client';
+
+declare global {
+  var prisma: PrismaClient | undefined;
+}
+
+let prisma: PrismaClient;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)